std-bits/base/src/main/java/org/enso/base/Array_Utils.java

```diff
@@ -0,0 +1,7 @@
+package org.enso.base;
+
+public class Array_Utils {
+  public static byte[] toByteArray(byte[] input) {
+    return input;
```

Comment:
the usage of `toByteArray` looks ... redundant? it's an identity function or what am I missing?

Reply:
forces the polyglot conversion of the Enso array to a byte array and will hence error early if it fails.

Reply:
Isn't this more of an assert then?

Reply:
renamed function.